dm/syncer/optimist.go

```diff
@@ -144,18 +144,20 @@ func (s *Syncer) handleQueryEventOptimistic(qec *queryEventContext) error {
 	case *ast.DropDatabaseStmt:
 		skipOp = true
 	case *ast.CreateTableStmt:
-		info.TableInfoBefore = tiAfter // for `CREATE TABLE`, we use tiAfter as tiBefore.
-		rev, err = s.optimist.PutInfoAddTable(info)
-		if err != nil {
+		// need to execute the DDL to the downstream, but do not do the coordination with DM-master.
+		op.DDLs = qec.needHandleDDLs
+		skipOp = true
```

Comment:
What if this table's table structure is different from other shard tables?

Reply:
if the new table is smaller than other shard tables, the task will run well. Here is the case

Reply:
What if there is a conflict?

Reply:
Conflict is not support. Even if we put info for that 'create table statement', the normal table is still a conflict table, just like we create a task with conflict tables.